Makefile

```
@@ -124,6 +124,14 @@ ifeq ($(NIM_PARAMS),)
 # The `git reset ...` will try to fix a `make update` that was interrupted
 # with Ctrl+C after deleting the working copy and before getting a chance to
 # restore it in $(BUILD_SYSTEM_DIR).
+ifeq (, $(shell which git-lfs))
+ifeq ($(shell uname), Darwin)
+$(error Git LFS not installed. Run 'brew install git-lfs' to set up.)
+else
+$(error Git LFS not installed.)
```

```suggestion
$(error Git LFS not installed. Run 'brew install git-lfs' to set up)
else
$(error Git LFS not installed)
```

In the error message there is already a trailing `.`